srcjs/init_shiny.js

```diff
@@ -414,7 +414,14 @@ function initShiny() {
   initialValues['.clientdata_url_hostname'] = window.location.hostname;
   initialValues['.clientdata_url_port']     = window.location.port;
   initialValues['.clientdata_url_pathname'] = window.location.pathname;
+
+  // Send initial URL search (query string) and update it if it changes
   initialValues['.clientdata_url_search']   = window.location.search;
+
+  $(window).bind("popstate", function() {
```

Comment:
I think this should be `.on` instead of `.bind`. See: http://api.jquery.com/bind/